schema/2.0.0/corpora_schema.md

```diff
@@ -670,6 +674,26 @@ When a dataset is uploaded, the cellxgene Data Portal MUST automatically add the
 </tbody></table>
 <br>
 
+### feature_reference
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>feature_reference</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Data Portal</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>str</code>. This MUST indicate the reference organism of genes, it MUST be <code>NCBITaxon:9606</code> for <i>Homo sapiens</i>, <code>NCBITaxon:10090</code> for <i>Mus musculus</i>, or <code>NCBITaxon:2697049</code> for  SARS-CoV-2.
```

Comment:
This MUST indicate the reference organism of genes. It MUST be ...

Reply:
Fixed